### PR TITLE
ECT-1327 Fix vulnerabilities

### PR DESCRIPTION
**Pull request by mmalek-sa**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -29,5 +29,6 @@ RUN chown -R arculix:arculix /app
 
 USER arculix
 
-EXPOSE 3000
+EXPOSE $PORT
 CMD [ "npm", "start"]
+HEALTHCHECK --interval=30s --timeout=30s --start-period=30s --retries=10 CMD netstat -an | grep $PORT > /dev/null; if [ 0 != $? ]; then exit 1; fi;
```
Comment (nhumble-sa) [inline Dockerfile:35]: Where does `PORT` come from? I don't see an `ARG` declaring it.

**Review by arolling-sa — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (nhumble-sa) [reply Dockerfile:35]: When I check the built image, it's not exposing any ports. Does the environment get set by docker-compose during the build too? I'm a little concerned this image breaks if it's not given this PORT.

**Review by nhumble-sa — APPROVED** [review]
👍 LGTM

File: Dockerfile (hunk 1 above)
Comment (mmalek-sa) [reply Dockerfile:35]: Added an `ARG` for `PORT`.

**Pull request merged**